src/Modules/Reputation/ReputationModule.sol

```diff
@@ -0,0 +1,309 @@
+     */
+    function freezeReputation(
+          address _account
+        , uint256 _frozenUntilEpoch
+    ) 
+        override
+        public
+    {
+        _freezeReputation(_callerReputationToken(), _account, _frozenUntilEpoch);
+    }
+
+    function _freezeReputation(
+          address _reputationToken
+        , address _account
+        , uint256 _frozenUntilEpoch
+    )
+        internal
+    {
+        ReputationTokenInterface(_reputationToken).freezeReputation(
+              _account
+            , _frozenUntilEpoch
+        );
+    }
+
+    /**
+     * @dev See {ReputationToken-lockReputation}.
+     */
+    function lockReputation(
+          address _account
+        , uint256 _amount
+    ) 
+        override
+        public
+    {
+        _lockReputation(_callerReputationToken(), _account, _amount);
+    }
+
+    function _lockReputation(
+          address _reputationToken
+        , address _account
+        , uint256 _amount
+    ) 
+        internal
+    {
+        ReputationTokenInterface(_reputationToken).lockReputation(
+              _account
+            , _amount
+        );
+    }
+
+    /**
+     * @dev See {ReputationToken-unlockReputation}.
+     */
+    function unlockReputation(
+          address _account
+        , uint256 _amount
+    ) 
+        override
+        public
+    {
+        _unlockReputation(_callerReputationToken(), _account, _amount);
+    }
+    
+    function _unlockReputation(
+          address _reputationToken
+        , address _account
+        , uint256 _amount
+    ) 
+        internal
+    {
+        ReputationTokenInterface(_reputationToken).unlockReputation(
+              _account
+            , _amount
+        );
+    }
+
+    /**
+     * @dev See {ReputationToken-getAvailableReputation}.
+     */
+    function getAvailableReputation(
+          address _laborMarket
+        , address _account
+    )
+        override
+        public
+        view
+        returns (
+            uint256
+        )
+    {
+        return ReputationTokenInterface(
+            getReputationToken(_laborMarket)
+        ).getAvailableReputation(_account);
+    }
+
+    function getPendingDecay(
+          address _laborMarket
+        , address _account
+    )
+        override
+        public
+        view
+        returns (
+            uint256
+        )
+    {
+        return ReputationTokenInterface(
+            getReputationToken(_laborMarket)
+        ).getPendingDecay(_account);
+    }
+
+    /**
+     * @dev See {ReputationToken-getReputationAccountInfo}.
+     */
+    function getReputationAccountInfo(
+          address _laborMarket
+        , address _account
+    )
+        override
+        public
+        view
+        returns (
+            ReputationTokenInterface.ReputationAccountInfo memory
+        )
+    {
+        return ReputationTokenInterface(
+            getReputationToken(_laborMarket)
+        ).getReputationAccountInfo(_account);
+    }
+
+    /**
```

Comment:
Looking for opinion: It's nice that we have these. Why should we include these? Could/should this be surfaced as a single object? 

Typically I've only ever seen getters for states that have packed values. Please teach me!

Reply:
Mmm I don't have a good answer. Original thinking was since these are used so much internally and independent of one another that it'd be more legible if they were their own getters. Happy to change to single obj though as I don't see why I thought this was necessary other than that